bin/run_eliot_web.sh

```diff
@@ -9,6 +9,8 @@
 : "${ELIOT_GUNICORN_WORKERS:=1}"
 : "${ELIOT_GUNICORN_TIMEOUT:=300}"
 : "${ELIOT_GUNICORN_CMD_PREFIX:=}"
+: "${ELIOT_GUNICORN_MAX_REQUESTS:=10000}"
+: "${ELIOT_GUNICORN_MAX_REQUESTS_JITTER:=1000}"
```

Comment:
These are the same values we use in the Socorro collector (antenna). We can tune it going forward.